Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/CertificateManagerServiceImpl.java

```diff
@@ -123,12 +123,7 @@ public String getTrustStoreLocation() {
 
     @Override
     public List<CertificateDTO> refreshKeyStores() throws CertificateManagerException {
-        try {
-            return listTrustStore(NhincConstants.ADMIN_CERT_LIST_KEYSTORE);
-        } catch (Exception e) {
-            LOG.error(UNABLE_TO_GET_CERTIFICATE, e.getLocalizedMessage(), e);
-            throw new CertificateManagerException(UNABLE_TO_GET_CERTIFICATE);
-        }
+        throw new UnsupportedOperationException("Not supported yet.");
```

Comment:
Instead of UnsupportedOperationException, can throw CertificateManagerException with a not supported yet operation msg. (optional)